journal-of-international-economic-law.csl

```diff
@@ -0,0 +1,405 @@
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <style-options punctuation-in-quote="false"/>
+    <terms>
+      <term name="et-al">et al</term>
+      <term name="ordinal">th</term>
+      <term name="ordinal-01">st</term>
+      <term name="ordinal-02">d</term>
+      <term name="ordinal-03">d</term>
+      <term name="ordinal-11">th</term>
+      <term name="ordinal-12">th</term>
+      <term name="ordinal-13">th</term>
+      <term name="paragraph" form="short">
+        <single>para</single>
+        <multiple>paras</multiple>
+      </term>
+      <term name="accessed">visited</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" delimiter-precedes-last="never"/>
+      <et-al term="et-al"/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="container-title">
+    <text variable="container-title" form="short" strip-periods="true"/>
+  </macro>
+  <macro name="author-bib">
+    <names variable="author">
+      <name and="text" name-as-sort-order="all"/>
+      <substitute>
+        <names variable="editor">
+          <name and="text" name-as-sort-order="all"/>
+        </names>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-note">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never"/>
+      <substitute>
+        <text macro="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="internet-location">
+    <choose>
+      <if variable="URL">
+        <text variable="URL" prefix=" "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="issued-long">
+    <text variable="volume"/>
+    <date delimiter=" " variable="issued">
+      <date-part name="day"/>
+      <date-part name="month" form="long"/>
+      <date-part name="year" form="long"/>
+    </date>
+  </macro>
+  <macro name="edition">
+    <number variable="edition" form="ordinal" text-case="lowercase" suffix=" "/>
+    <choose>
+      <if match="any" is-numeric="edition">
+        <text term="edition" form="short" strip-periods="true" suffix="."/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="sort-by-type">
+    <choose>
+      <if type="bill legislation" match="any">
+        <text value="1"/>
+      </if>
+      <else-if type="legal_case">
+        <text value="2"/>
+      </else-if>
+      <else-if type="book thesis" match="any">
+        <text value="3"/>
+      </else-if>
+      <else-if type="article-journal chapter article-newspaper" match="any">
+        <text value="4"/>
+      </else-if>
+      <else>
+        <text value="5"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="render-chapter">
+    <group>
+      <text variable="title" strip-periods="true" quotes="false" prefix="‘" suffix="’, "/>
```

Comment:
don't hard-code quotes. Instead use `quotes="true"` with en-GB locale, those default to single quotes.